bigquery/README.rst

```diff
@@ -48,39 +48,49 @@ Load data from CSV
     import csv
 
     from google.cloud import bigquery
+    from google.cloud.bigquery import Dataset
+    from google.cloud.bigquery import LoadJobConfig
     from google.cloud.bigquery import SchemaField
 
     client = bigquery.Client()
 
-    dataset = client.dataset('dataset_name')
-    dataset.create()  # API request
+    dataset_ref = client.dataset('dataset_name')
+    dataset = Dataset(dataset_ref)
+    dataset.description = 'my dataset'
+    dataset = client.create_dataset(dataset)  # API request
 
     SCHEMA = [
         SchemaField('full_name', 'STRING', mode='required'),
         SchemaField('age', 'INTEGER', mode='required'),
     ]
-    table = dataset.table('table_name', SCHEMA)
-    table.create()
+    table = dataset_ref.table('table_name')
```

Comment:
table_ref?

Reply:
Yeah, good idea. Done.

This sample is a bit big for my taste, so I also split the dataset creation to its own sample.